.github/workflows/test.yml

```diff
@@ -29,6 +29,17 @@ jobs:
           brew install docker
           colima start
 
+      - name: Install protobuf Ubuntu
+        if: startsWith(matrix.os, 'ubuntu-')
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y protobuf-compiler
+
+      - name: Install protobuf MacOS
+        if: startsWith(matrix.os, 'macos-')
+        run: |
+          brew install protobuf
```

Comment:
@NathanFlurry curious on your thoughts for this. If we leave it in, then I'll need to add it to the release build I think.

Reply:
Resolved by #147